ionoscloud/resource_ipfailover.go

```diff
@@ -98,19 +98,22 @@ func resourceLanIPFailoverRead(ctx context.Context, d *schema.ResourceData, meta
 		return diags
 	}
 
-	if lan.Properties.IpFailover != nil {
-		err := d.Set("ip", *(*lan.Properties.IpFailover)[0].Ip)
-		if err != nil {
-			diags := diag.FromErr(fmt.Errorf("error while setting ip property for IpFailover %s: %w", d.Id(), err))
-			return diags
+	failoverSlice := lan.Properties.IpFailover
+	if lan.Properties != nil && lan.Properties.IpFailover != nil && len(*failoverSlice) > 0 {
```

Comment:
why don't you verify here failoverSlice != nil instead of lan.Properties.IpFailover != nil?